Makefile

```diff
@@ -28,9 +28,15 @@ build:
 	rm -rf ./dist
 	hugo -s site --verbose --destination ../dist/html --printUnusedTemplates --printPathWarnings --gc
 
+
+.PHONY: test
+test:
+	npm install
+	npm run test-html
+
 .PHONY: compass-icons
 compass-icons:
 	mkdir -p site/static/css
 	mkdir -p site/static/font
 	curl --no-progress-meter -o site/static/css/compass-icons.css https://mattermost.github.io/compass-icons/css/compass-icons.css
-	curl --no-progress-meter -o "site/static/font/compass-icons.#1" "https://mattermost.github.io/compass-icons/font/compass-icons.{eot,woff2,woff,ttf,svg}"
+	curl --no-progress-meter -o "site/static/font/compass-icons.#1" "https://mattermost.github.io/compass-icons/font/compass-icons.{eot,woff2,woff,ttf,svg}"
```

Comment:
Missing linebreak

Reply:
I can't figure out how to change that 😓
Adding a linebreak will introduce a new line 🤔